operations/commit_queue.go

```diff
@@ -156,7 +156,7 @@ func mergeCommand() cli.Command {
 				commits:     commits,
 				id:          c.String(resumeFlagName),
 				pause:       c.Bool(pauseFlagName),
-				skipConfirm: c.Bool(yesFlagName),
+				skipConfirm: c.Bool(yesFlagName) || c.Bool(skipConfirmFlagName),
```

Comment:
I think you don't need to check yes and skipConfirm since you join the flag names in `addSkipConfirmFlag`, you could just check skipConfirmFlagName (I'm not 100% sure about this though so I would verify it locally)

Reply:
Tested locally and you are right - joining the flag names was all that was required. Fixed in latest commit.